File: Minah7/week01/숫자의_표현.java
// 성공

class Solution {
    public int solution (int n) {
        int answer = 0;
        int start = 1;
        while (start <= n) {
            int sum = start;
            for(int i = start + 1; i <= n; i++) {
                if(sum == n) {
                    System.out.println(start);
                    answer++;
                    break;
                } 
                sum += i;
                if(sum > n) {
                    break;
                }
            }
            start++;
        }
        answer++;
        return answer;
    }
}

File: damdam6/week08/숫자의 표현.js
function solution(n) {
    var answer = 1;
    
    for(let i=1;i<n/2;i++){
        let result = 0;
        let up = i;
        while(result < n){
            result += up;
            up++;           
            
        }
        
        if(n==result){
            // console.log(i);
            answer++;
        }
    }
    
    
    return answer;
}